src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
Classes should have an access modifier, preferably `public` if it's meant to be accessible from outside its package.